BUILD

```diff
@@ -1,4 +1,9 @@
-alias(
-    name = "bazel-diff",
-    actual = "//src/main/java/com/bazel_diff:bazel-diff"
-)
+#alias(
```

Comment:
Feel free to delete, I do not use these aliases

Reply:
sure, will create another PR tomorrow

Reply:
https://github.com/Tinder/bazel-diff/pull/127